### ISSUE TITLE
Consider sending X-Frame-Options and CSP headers in authorization form responses

### ISSUE THREAD

**Issue by barnabywalters** [Taproot/indieauth#21]
Per https://www.oauth.com/oauth2-servers/authorization/security-considerations/ — the auth form (and any other user-facing auth screens which it redirects to) should not allow themselves to be embedded in a frame.

**Comment by barnabywalters** [comment]
According to https://infosec.mozilla.org/guidelines/web_security#x-frame-options, these options would prevent framing

```
Content-Security-Policy: frame-ancestors 'none'
X-Frame-Options: DENY
```

**Comment by barnabywalters** [comment]
Fixed in f0a5e84, additionally fixed Cache-Control and added Pragma headers where relevant
